stock_move_actual_date/tests/test_stock_move_actual_date.py

```diff
@@ -0,0 +1,142 @@
+# Copyright 2024 Quartile (https://www.quartile.co)
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from datetime import date
+
+from freezegun import freeze_time
+
+from odoo.tests.common import TransactionCase
+
+
+class TestStockMoveActualDate(TransactionCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        cls.partner = cls.env["res.partner"].create({"name": "Test Partner"})
+        product_category = cls.env["product.category"].create(
+            {
+                "name": "Test Category",
+                "property_cost_method": "fifo",
+                "property_valuation": "real_time",
+            }
+        )
+        cls.product_1 = cls.env["product.product"].create(
+            {
+                "name": "Test Product",
+                "type": "product",
+                "categ_id": product_category.id,
+            }
+        )
+        cls.supplier_location = cls.env.ref("stock.stock_location_suppliers")
+        cls.stock_location = cls.env.ref("stock.stock_location_stock")
+        cls.env.user.tz = "Asia/Tokyo"
```

Comment:
For clarity of the test scenario, it would be better to put this inside `test_stock_move_without_actual_date_from_picking_or_scrap()` which is the only test this setting matters.